app-backend/project/plugins.sbt

```diff
@@ -3,22 +3,20 @@ resolvers ++= Seq(
   Opts.resolver.sonatypeReleases
 )
 
-resolvers += Classpaths.typesafeResolver
-
 resolvers += Resolver.bintrayIvyRepo("kamon-io", "sbt-plugins")
 
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
+addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.1")
 
-addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.10")
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.3")
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.4")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 
-addSbtPlugin("com.sksamuel.scapegoat" %% "sbt-scapegoat" % "1.0.4")
+addSbtPlugin("com.sksamuel.scapegoat" %% "sbt-scapegoat" % "1.0.7")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0")
 
-addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.0")
+addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.2")
 
-addSbtPlugin("io.kamon" % "sbt-aspectj-runner" % "1.0.1")
+// addSbtPlugin("io.kamon" % "sbt-aspectj-runner" % "1.0.1")
```

Comment:
@fosskers was there a reason this is commented?

Reply:
Yes, and that's something I wanted to talk to you about.

[The plugin](https://github.com/kamon-io/sbt-aspectj-runner) seems quite old, and meant for [Play Framework](https://playframework.com/) projects. It wasn't clear to me what purpose it was serving in RF, and it was the only thing that has no SBT 1.0.x compatibility (thus breaking builds if we want the new hotness).

As a side-contributor to RF I obviously can't just willy-nilly disable plugins on my own, so for the time being I commented it out.